operator/src/api/preview/scheduledbackup_types.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package preview
+
+import (
+	cnpgv1 "github.com/cloudnative-pg/cloudnative-pg/api/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ScheduledBackupSpec defines the desired state of ScheduledBackup
+type ScheduledBackupSpec struct {
```

Comment:
Why cant we combine this with normal BackupSpec? Is CNPG doing this in two different ways?